### ISSUE TITLE
Socket Hangup when trying PUT to create

### ISSUE THREAD

**Issue by michielbdejong** [pdsinterop/community-server#4]
https://github.com/michielbdejong/community-server/runs/1862991873#step:13:491

This causes multiple tests to fail for "PUT to create"

**Comment by ylebre** [comment]
It looks like the PUT request fails because the test suite isn't passing a body to put - when running locally:
```
Creating a folder fails because of lacking body:
curl -X PUT -H "Content-Type: text/turtle" -H "Link: '<http://www.w3.org/ns/ldp#BasicContainer>; rel=\"type\"'" http://localhost:3000/testfolder/
UnsupportedHttpError: PUT operations require a body.
```

**Comment by michielbdejong** [comment]
Investigating. 

Btw, let's use the `v0.7.0` git tag of CSS because there are [known bugs](https://github.com/solid/community-server/issues/599#issuecomment-776537782) in the current `main` branch.

**Comment by michielbdejong** [comment]
Rebased on `v0.7.0` instead of on latest upstream `main` in https://github.com/michielbdejong/community-server/pull/1 now, let's see how that goes.

**Comment by michielbdejong** [comment]
Make sure you `npm ci` after `git pull`

**Comment by michielbdejong** [comment]
It's weird, the tests pass on my localhost but not on GitHub actions. See https://github.com/michielbdejong/community-server/issues/11

**Comment by michielbdejong** [comment]
Confirmed, this was fixed by @joachimvh in https://github.com/solid/community-server/issues/599
Thanks, Joachim!

So I was now able to merge the latest main branch including this fix into https://github.com/michielbdejong/community-server/pull/1 and the Socket Hangup is fixed.

Still weird that this wasn't picked up by their integration tests. The CSS code was clearly broken, but all tests were green.

I opened an issue about that but somebody closed it. I asked for it to be reopened but that request was marked as spam by the same community member.
